lib/index.d.ts

```diff
@@ -81,15 +81,16 @@ declare namespace boom {
     }
 }
 
+interface BoomConstructor {
+    new <Data = any, Decoration extends object = object>(message?: string | Error, options?: boom.Options<Data, Decoration>): Boom<Data> & Decoration;
```

Comment:
While working on this I was wondering whether we should default on generic parameters, I'm not sure it's that useful.

Reply:
I don't know, really. I just tried to copy it as-is, though I added `extends object` to signal that it only works with objects.